README.md

```diff
@@ -37,8 +37,29 @@ User: A person that uses our software.
 
 Cantis also generates its own [glossary](glossary.txt).
 
+## Install
+In order to use the `@GlossaryTerm` annotation, you must add a dependency to `cantis` to your project:
+
+Maven:
+
+```xml
+<dependency>
+    <groupId>com.github.korthout</groupId>
+    <artifactId>cantis</artifactId>
+    <version>0.1</version>
+</dependency>
+```
+
+Gradle:
+
+```
+dependencies {
+    implementation 'org.hibernate:hibernate-core:3.6.7.Final'
+}
+```
+
 ## Usage
-Adding Cantis to your own project is easy. Simply:
+Using Cantis to your own project is easy. Simply:
```

Comment:
I think I understand what you wanted to achieve here. With the installation part, Cantis has already been added to your own project. But in that case it should be something like:

```suggestion
Using Cantis on your own project is easy. Simply:
```

Or: 
```suggestion
Applying Cantis to your own project is easy. Simply:
```